src/midcom/datamanager/extension/type/select.php

```diff
@@ -35,18 +36,30 @@ public function configureOptions(OptionsResolver $resolver)
 
         $map_options = function (Options $options) {
             $return_options = array();
+
             if (isset($options['type_config']['options'])) {
                 foreach ($options['type_config']['options'] as $key => $value) {
-                    //symfony expects only strings
+                    //symfony expects only string
                     $return_options[$value] = (string)$key;
                 }
                 return $return_options;
             }
         };
 
+        $multiple_options = function (Options $options)
+        {
+            $return_options = array();
+
+            if(isset($options['type_config']['allow_multiple'])){
```

Comment:
btw `isset` only checks whether or not a variable exists, so if `allow_multiple` is set to `false`, this funtion here would still return `true`